Translations/pt_BR.po

```diff
@@ -4805,335 +4809,335 @@ msgstr ""
 
 #: Source/monstdat.cpp:482
 msgid "Bonehead Keenaxe"
-msgstr ""
+msgstr "Osteocéfalo Machado"
 
 #: Source/monstdat.cpp:483
 msgid "Bladeskin the Slasher"
-msgstr ""
+msgstr "Laminiderme, o Esfaqueador"
 
 #: Source/monstdat.cpp:484
 msgid "Soulpus"
-msgstr ""
+msgstr "Pus de Alma"
 
 #: Source/monstdat.cpp:485
 msgid "Pukerat the Unclean"
-msgstr ""
+msgstr "Vomirrato, o Impuro"
 
 #: Source/monstdat.cpp:486
 msgid "Boneripper"
-msgstr ""
+msgstr "Estripa-ossos"
 
 #: Source/monstdat.cpp:487
 msgid "Rotfeast the Hungry"
-msgstr ""
+msgstr "Putrefórico, o Faminto"
 
 #: Source/monstdat.cpp:488
 msgid "Gutshank the Quick"
-msgstr ""
+msgstr "Visceraguda o Veloz"
 
 #: Source/monstdat.cpp:489
 msgid "Brokenhead Bangshield"
-msgstr ""
+msgstr "Traumocéfalo Bate-Escudos"
 
 #: Source/monstdat.cpp:491
 msgid "Rotcarnage"
-msgstr ""
+msgstr "Putrificina"
 
 #: Source/monstdat.cpp:492
 msgid "Shadowbite"
-msgstr ""
+msgstr "Ferrão das Sombras"
 
 #: Source/monstdat.cpp:493
 msgid "Deadeye"
-msgstr ""
+msgstr "Olho Morto"
 
 #: Source/monstdat.cpp:494
 msgid "Madeye the Dead"
-msgstr ""
+msgstr "Olho Louco, o Morto"
 
 #: Source/monstdat.cpp:496
 msgid "Skullfire"
-msgstr ""
+msgstr "Fogo Cranial"
 
 #: Source/monstdat.cpp:497
 msgid "Warpskull"
-msgstr ""
+msgstr "Crânio Torto"
 
 #: Source/monstdat.cpp:498
 msgid "Goretongue"
-msgstr ""
+msgstr "Sanguinolíngue"
 
 #: Source/monstdat.cpp:499
 msgid "Pulsecrawler"
-msgstr ""
+msgstr "Pulsejador"
 
 #: Source/monstdat.cpp:500
 msgid "Moonbender"
-msgstr ""
+msgstr "Verga-Luas"
 
 #: Source/monstdat.cpp:501
 msgid "Wrathraven"
-msgstr ""
+msgstr "Corvo da Cólera"
 
 #: Source/monstdat.cpp:502
 msgid "Spineeater"
-msgstr ""
+msgstr "Come-Espinhas"
 
 #: Source/monstdat.cpp:503
 msgid "Blackash the Burning"
-msgstr ""
+msgstr "Víbice Negro, o Flamejante"
```

Comment:
I'm curious where this came from. The translation makes zero sense to me. Isn't the "ash" in the original related to actual ash?
```suggestion
msgstr "Cinzas Negras, o Flamejante"
```